src/gui/windows/debug.zig

```diff
@@ -25,7 +25,14 @@ pub var window = GuiWindow {
 pub fn render() void {
 	draw.setColor(0xffffffff);
 	var y: f32 = 0;
-	draw.print("    fps: {d:.0} Hz{s}", .{1.0/main.lastFrameTime.load(.monotonic), if(main.settings.vsync) @as([]const u8, " (vsync)") else ""}, 0, y, 8, .left);
+	const fpsCapText = if(main.settings.fpsCap) |fpsCap| std.fmt.allocPrint(main.stackAllocator.allocator, " (limit: {d:.0} Hz)", .{fpsCap}) catch unreachable else null;
```

Comment:
You can do `else ""`
freeing slices of length zero is explicitly allowed by the allocator interface.
Then you can just do `defer main.stackAllocator.free(fpsCapText)`

Reply:
Wouldn't that try to free a global string?

Reply:
I see. https://ziglang.org/documentation/master/std/#std.mem.Allocator.free. Updated, hopefully they won't change this too soon then.

Reply:
Nah, they won't change it. Many data structures in the standard library already depend on it.